### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by Luna5-threshold**
Creating the security policy: reporting a vulnerability for threshold network. This text is based on the nucypher policy, please check if you want to add or modify anything to adjust better to this project.

### CODE REVIEW

File: SECURITY.md
Comment (derekpierre) [inline SECURITY.md:5]:
```suggestion
If you identify vulnerabilities with any Threshold Network code, please email security@threshold.network with relevant information to your findings. We will work with researchers to coordinate vulnerability disclosure between our stakers, partners, and users to ensure the successful mitigation of vulnerabilities.
```

File: SECURITY.md
Comment (derekpierre) [inline SECURITY.md:9]:
```suggestion
Sometimes vulnerabilities are more sensitive in nature and require extra precautions. We are happy to work together to use a more secure medium, such as Signal. Email security@threshold.network and we will coordinate a communication channel that we're both comfortable with.
```

File: SECURITY.md
Comment (derekpierre) [inline SECURITY.md:11]:
```suggestion
A great place to begin your research is by working on our testnet. Please see our [documentation](https://docs.threshold.network) to get started. We ask that you please respect network machines and their owners. If you find a vulnerability that you suspect has given you access to a machine against the owner's permission, stop what you're doing and immediately email security@threshold.network.
```

**Review by derekpierre — COMMENTED** [review]
Minor comments. Things that weren't fixed in the `nucypher` policy.

**Review by derekpierre — DISMISSED** [review]
🎸 

Side note: we should also add something to https://docs.threshold.network if we haven't already.

File: SECURITY.md
Comment (cygnusv) [inline SECURITY.md:5]:
```suggestion
If you identify vulnerabilities with any Threshold Network code, please email `security@threshold.network` with relevant information to your findings. We will work with researchers to coordinate vulnerability disclosure between our stakers, partners, and users to ensure the successful mitigation of vulnerabilities.
```

File: SECURITY.md
Comment (cygnusv) [inline SECURITY.md:11]:
```suggestion
A great place to begin your research is by working on our testnet. Please see our [documentation](https://docs.threshold.network) to get started. We ask that you please respect network machines and their owners. If you find a vulnerability that you suspect has given you access to a machine against the owner's permission, stop what you're doing and immediately email `security@threshold.network`.
```

**Review by cygnusv — DISMISSED** [review]

**Comment by Luna5-threshold** [comment]
> 

I have created a new section in the docs, containing the very same text, and it is waiting for review and approval. I am introducing the fixed version of this text in a few minutes there, so you can verify and merge.

**Pull request merged**
